operations/fetch.go

```diff
@@ -502,7 +502,7 @@ func downloadUrls(root string, urls chan artifactDownload, workers int) error {
 				// If we can get the info, determine the file size so that the human can get an
 				// idea of how long the file might take to download.
 				// TODO: progress bars.
-				length, _ := strconv.Atoi(resp.Header.Get("Content-Length"))
+				length, _ := strconv.Atoi(resp.Header.Get(evergreen.ContentLengthHeader))
```

Comment:
why not resp.ContentLength?

Reply:
done

Reply:
I bet this code predated that. 

Not sure that it matters, but I'm inclined to not change it, at this point.